src/server/scripts/Pet/pet_dk.cpp

```diff
@@ -74,9 +68,28 @@ class npc_pet_dk_ebon_gargoyle : public CreatureScript
                     owner->RemoveAurasDueToSpell(SPELL_DK_SUMMON_GARGOYLE_2);
             }
 
+            void IsInteruptSpell(SpellInfo const* spell)
+            {
+                if (spell->HasEffect(SPELL_EFFECT_INTERRUPT_CAST))
```

Comment:
why?
SpellHistory already handle with it.

Reply:
This script 2015 year, i say help me move new standart.